common/buildings/wc_culture_group.txt

```diff
@@ -0,0 +1,172 @@
+#Created on 9/29/2019
+#Modified on 9/29/2019 by DukeJeans
+
+castle = {
+    ## Dryad Group ##
+
+    wcca_dryad_1 = {
+        desc = wcca_dryad_desc
+        potential = {
+            culture_group = dryad_group
+        }
+		trigger = { 
+			TECH_CASTLE_CONSTRUCTION = 0
+			ROOT = {
+				culture_group = dryad_group
+			}
+		}
+        prerequisites = { ca_wall_2 }
+		gold_cost = 200
+		build_time = 730
+
+        horse_archers = 25
+
+        horse_archers_morale = 0.10
+        horse_archers_offensive = 0.05
+    }
+    wcca_dryad_2 = {
+        desc = wcca_dryad_desc
+        potential = {
+            culture_group = dryad_group
+        }
+		trigger = { 
+			TECH_CASTLE_CONSTRUCTION = 1
+			ROOT = {
+				culture_group = dryad_group
+			}
+		}
+		gold_cost = 300
+		build_time = 1095
+
+        horse_archers = 5
```

Comment:
> Troop levy equals 60 gold in maintenance, plus 20 per building level,

It means that you get +60/80/100/120 levy/maintenance.

if 1 horse_archers cost 4, tier-4 building must give you 15/20/25/30 (90 total) horse_archers

Reply:
Thanks for the correction ❤️